includes/class-wc-calypso-bridge-woocommerce-admin-features.php

```diff
@@ -100,6 +101,15 @@ public function initialize() {
 		add_filter( 'pre_option_woocommerce_navigation_enabled', static function ( $pre ) {
 			return 'no';
 		}, PHP_INT_MAX );
+
+		/*
+		 * Suppress WooCommerce Help tab and move onboarding reset settings under 'Settings > General > Onboarding'.
```

Comment:
@manospsyx Should we maybe limit this to Woo Express only? `wc_calypso_bridge_is_woo_express_plan` ?

Reply:
> No, because the Ecommerce Plan is still a paid plan.
> We shouldn't be pointing to the free .org forum for support there